Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.h

```diff
@@ -407,6 +408,9 @@ namespace AzToolsFramework
             // Used for loading/saving Prefab Template files.
             PrefabLoader m_prefabLoader;
 
+            //Used for loading/saving Prefab View Bookmarks.
+            ViewBookmarkLoader m_viewBookmarkLoader;
+
```

Comment:
I don't think this system should be part of the Prefabs class. It's pretty much just a standalone component and a system component handling it, so it's more of an Editor system than anything.
Is there a technical reason why you need it there? (It doesn't seem to be using any of the Prefabs API directly, so I guess not?)

Else I'd rather see it moved to be a standalone system component.

Reply:
Thanks for changing the namespace :) Realized I didn't say it in the comment above, but I'd also just move the file out of the Prefab folder into its own folder in AzToolsFramework (I'm thinking Viewport? Or its own thing maybe).

Reply:
Yeah Viewport folder sounds like a good plan! 👍

Reply:
Just moved everything over to the Viewport folder and created a System Component for the ViewBookmarks! 👍